test/com/moclojer/middleware/latency_test.clj

```diff
@@ -33,9 +38,11 @@
   (testing "Timeout simulation"
     (let [handler (fn [_] {:status 200 :body "OK"})
           wrapped-handler (latency/wrap-latency handler)
+          timeout-ms 100
+          buffer-ms 10  ;; Buffer para acomodar variações do sistema
           request {:reitit.core/match
-                  {:data {:latency {:timeout-rate 1.0  ;; Always timeout
-                                  :timeout-ms 100}}}}
+                   {:data {:latency {:timeout-rate 1.0  ;; Always timeout
+                                     :timeout-ms timeout-ms}}}}
```

Comment:
here too. Clojure resource is scarce. It would take me ages to find where you took this from.